integration-tests/hibernate-orm-panache/src/test/java/io/quarkus/it/panache/PanacheFunctionalityTest.java

```diff
@@ -76,11 +76,15 @@ public void testJaxbAnnotationTransfer() {
     }
 
     /**
-     * This test is disabled in native mode as there is no interaction with the quarkus integration test endpoint.
+     * _PanacheEntityBase_ has the method _isPersistent_. This method is used by jackson to serialize the attribute *persistent*
+     * in the JSON which is not intended. This test ensures that there something done that this does not happen.
```

Comment:
> This test ensures that there something done that this does not happen

This doesn't seem a valid sentence.

Reply:
"This test ensures that the attribute *persistent* is not generated when using Jackson"

Better?

Reply:
Yes.

Reply:
I will adapt the PR next week. At the moment I do not have access to a my workstation...